Core/Rewriters/MoqSyntaxFactory.cs

```diff
@@ -292,6 +297,17 @@ public static ObjectCreationExpressionSyntax StrictMockCreationExpression (TypeA
                     SyntaxFactory.IdentifierName (identifierName),
                     SyntaxFactory.IdentifierName ("Verify"))));
 
+    public static ExpressionStatementSyntax VerifyStatement (IdentifierNameSyntax identifierName, ExpressionSyntax expression, Times times)
```

Comment:
I don't think the Times enum serves a purpose here, you could just pass in an `int`, `Times.Never` would be `0`.